SPECS/ig/ig.spec

```diff
@@ -64,6 +64,10 @@ fi
 %{_bindir}/ig
 
 %changelog
+* Mon Jul 01 2024 Francis Laniel <flaniel@linux.microsoft.com> - 0.30.0-1
```

Comment:
Just looked and saw that the `version:` field at the top of the spec hasn't been updated. That is also likely why the CI checks are still referring to `ig-0.29.0`